### PR TITLE
chore(scheduler-docs): add the View Switching Guide

### PR DESCRIPTION
**Pull request by MaximKudriavtsev**

### CODE REVIEW

File: packages/dx-react-scheduler/docs/guides/view-switching.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
+# React Scheduler - View Switching
+
+If the React Scheduler has more then one view, your can choose showed view by UI with `ViewSwitcher` plugin. The following guide provides information about navigation between available views.
```
Comment (ushkal) [inline packages/dx-react-scheduler/docs/guides/view-switching.md]: -> switch between views

File: packages/dx-react-scheduler/docs/guides/view-switching.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+### Uncontrolled Mode
+
+In the uncontrolled mode, specify the initial view name value in the `ViewState` plugin’s `defaultCurrentViewName` property.
+
+.embedded-demo({ "path": "scheduler-view-switcher/uncontrolled", "showThemeSelector": true })
+
+### Controlled Mode
+
+In the controlled mode, pass the view name option to the `ViewState` plugin’s `currentViewName` property and handle the `onCurrentViewNameChange` event to control the view state externally.
```
Comment (ushkal) [inline packages/dx-react-scheduler/docs/guides/view-switching.md]: view state -> view name ?

File: packages/dx-react-scheduler/docs/guides/view-switching.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+
+.embedded-demo({ "path": "scheduler-view-switcher/controlled", "showThemeSelector": true })
+
+## External Control
+
+To control the view state externally define the `ViewState` plugin's `currentViewName` property without adding the `ViewSwitching` plugin.
```
Comment (SergeyAlexeev) [inline packages/dx-react-scheduler/docs/guides/view-switching.md]: > the `ViewSwitching` plugin.

the `ViewSwitcher` plugin.

File: packages/dx-react-scheduler/docs/guides/view-switching.md [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+
+### Uncontrolled Mode
+
+In the uncontrolled mode, specify the initial view name value in the `ViewState` plugin’s `defaultCurrentViewName` property.
+
+.embedded-demo({ "path": "scheduler-view-switcher/uncontrolled", "showThemeSelector": true })
+
+### Controlled Mode
+
+In the controlled mode, pass the view name option to the `ViewState` plugin’s `currentViewName` property and handle the `onCurrentViewNameChange` event to control the view state externally.
```
Comment (SergeyAlexeev) [inline packages/dx-react-scheduler/docs/guides/view-switching.md]: > In the controlled mode, pass the view name option to the `ViewState` plugin’s `currentViewName` property

pass the view name **option** to ... **property**?

Comment (MaximKudriavtsev) [reply packages/dx-react-scheduler/docs/guides/view-switching.md]: @SergeyAlexeev, could I remove the `option` word?

Comment (SergeyAlexeev) [reply packages/dx-react-scheduler/docs/guides/view-switching.md]: I think so)

**Pull request merged**